package/MDAnalysis/analysis/helanal.py

```diff
@@ -117,20 +124,24 @@
 
 """
 from __future__ import print_function, division, absolute_import
-from six.moves import range, zip
-
 import os
 
 import numpy as np
 
 import MDAnalysis
-from MDAnalysis.lib.log import ProgressBar
-from MDAnalysis.lib import mdamath
+from ..lib.log import ProgressBar
+from ..lib import mdamath, util
 
 import warnings
 import logging
 logger = logging.getLogger("MDAnalysis.analysis.helanal")
 
+warnings.warn("This module is deprecated as of MDAnalysis version 1.0. "
+              "It will be removed in MDAnalysis version 2.0."
+              "Please use (available in 2.0.0) "
+              "MDAnalysis.analysis.helix_analysis instead.",
+              category=DeprecationWarning)
+
 def center(coordinates):
```

Comment:
These are all public facing functions, do we need to add deprecation warnings here too or do we just assume the warning on module importing will be sufficient?

Reply:
I think the module import warning is sufficient.